src/lib/mongoBackend/mongoConnectionPool.cpp

```diff
@@ -191,9 +193,14 @@ static DBClientBase* mongoConnect
 
   if (connected == false)
   {
-    LM_E(("Database Error (connection failed, after %d retries: '%s')", retries, err.c_str()));
+    char cV[64];
```

Comment:
The code looks like this:

    char cV[64];
    snprintf(cV, sizeof(cV), "connection failed, after %d retries", retries);
    std::string details = std::string(cV) + ": " + err;

So, that 64 is 'space enough for the coming sentence' ...
I really don't see how I could make a definition for each and every sentence that is used.